client/routes/signin/index.test.jsx

```
@@ -0,0 +1,86 @@
+// Mock stores
+jest.mock('../../stores/authStore');
+jest.mock('../../stores/settingsStore');
+// Mock any API calls if necessary, though for this test, not directly testing signIn API
+jest.mock('../../api/authentication', () => ({
+    signIn: jest.fn(),
+}));
+
+// Mock window.location.href
+const originalLocation = window.location;
+beforeAll(() => {
+    delete window.location;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)